### ISSUE TITLE
Hidden dependency: libarchive

### ISSUE THREAD

**Issue by TinoDidriksen** [divvun/libdivvun#9]
`./configure --enable-checker --enable-cgspell --enable-python-bindings` ran fine, but the build said:

````
In file included from checker.cpp:19:
In file included from ./pipeline.hpp:29:
./pipespec.hpp:50:10: fatal error: 'archive.h' file not found
````

`configure` needs to check for a usable `libarchive`.

**Comment by unhammer** [comment]
Hm, I did *try* to check ( https://github.com/divvun/libdivvun/blob/master/configure.ac#L108 and https://github.com/divvun/libdivvun/blob/master/configure.ac#L88 ). m4 strikes again.

**Comment by TinoDidriksen** [comment]
Ah, what probably happened then is that it saw libarchive from Macports but for whatever reason didn't get the -I passed along.

**Comment by unhammer** [comment]
Do you know which goal it was building when that happened? I see the `Makefile.am` only mentions libarchive for `divvun-gen-sh`, but I guess it should mention that elsewhere too.

```
divvun_gen_sh_LDADD    = $(PUGIXML_LIBS)   $(LIBARCHIVE_LIBS)
divvun_gen_sh_CXXFLAGS = $(PUGIXML_CFLAGS) $(LIBARCHIVE_CFLAGS)

```

**Comment by unhammer** [comment]
Was this fixed by #11 ?

**Comment by TinoDidriksen** [comment]
Hm, I guess that must have been the core problem. So yeah, fixed in https://github.com/divvun/libdivvun/commit/51aceee490ddf0f005df8295dcf1c73e7ca31a5f
